NuGet.config

```diff
@@ -11,5 +11,24 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
   </packageSources>
-  <disabledPackageSources />
+  <packageSourceMapping>
+    <!-- key value for <packageSource> should match key values from <packageSources> element -->
+    <packageSource key="dotnet-public">
+      <package pattern="Microsoft.DotNet.PlatformAbstractions" />
+      <package pattern="*" />
+    </packageSource>
+    <packageSource key="dotnet-eng">
+      <package pattern="Microsoft.DotNet.Arcade.Sdk" />
+      <package pattern="MicroBuild.Core.Sentinel" />
+      <package pattern="Microsoft.Signed.Wix" />
+      <package pattern="Microsoft.DotNet.Build.Tasks.Feed" />
+      <package pattern="Microsoft.DotNet.SignTool" />
+      <package pattern="Microsoft.DotNet.Tar" />
+      <package pattern="Microsoft.SymbolUploader.Build.Task" />
+      <package pattern="Microsoft.DotNet.*" />
+      <package pattern="Microsoft.DiaSymReader.*" />
+      <package pattern="Microsoft.SourceBuild.Intermediate.*" />
+      <package pattern="sn" />
+    </packageSource>
+  </packageSourceMapping>
```

Comment:
Package source mapping doesn't work with Maestro internal branches (i.e. release and servicing): https://github.com/dotnet/arcade-services/issues/3002

I recommend to do disable the warning: https://github.com/dotnet/format/blob/4425369d3706effc4f5aa67ec16828b07e69c713/Directory.Packages.props#L6-L7

Reply:
Looking at the builds, this looks insufficient so far, any other suggestions or changes?

Reply:
It is sufficient but this project incorrectly overwrites `NoWarn` globally: https://github.com/BenjaminMichaelis/command-line-api/blob/d0270ef03bd9ac9d518197d815f46d0733f35f33/src/System.CommandLine.DragonFruit.Tests/System.CommandLine.DragonFruit.Tests.csproj#L10

This should `<NoWarn>$(NoWarn);1701;1702;1591</NoWarn>` instead.

Reply:
Good spot, thanks